tests/wrap.py

```diff
@@ -34,25 +34,22 @@ def testMakeControl(self):
         config.foo = 2
         config.bar.append("baz")
         control = config.makeControl()
-        self.assert_(testLib.checkControl(control, config.foo, config.bar))
+        self.assert_(testLib.checkControl(control, config.foo, config.bar._list))
```

Comment:
We could of course make the present example slightly less ugly by adding a `.list()` method to the `List` class and have the method not be internal.